indexing-service/src/main/java/io/druid/indexing/common/actions/SegmentListUsedAction.java

```diff
@@ -53,9 +69,9 @@ public String getDataSource()
   }
 
   @JsonProperty
```

Comment:
ah nevermind, I thought you had removed the de-serialization support for interval as well.